src/openrct2/ride/Vehicle.cpp

```diff
@@ -3920,7 +3920,10 @@ void Vehicle::UpdateMotionBoatHire()
             }
             eax -= velocity;
             edx = powered_acceleration * 2;
-            ecx += (eax * edx) / ebx;
+            if (ebx != 0)
```

Comment:
Can't believe we still have such variable names, most of the code looks like its working out acceleration.

Reply:
Just saying, it can be done in this PR or another, either way works for me.